src/v/raft/consensus.h

```diff
@@ -543,6 +543,11 @@ class consensus {
         return _configuration_manager.has_configuration_override();
     }
 
+    void toggle_block_append_entries(bool block) {
```

Comment:
This method name doesn't susggest it is a fault injection mechanism. Can we add a comment on top of that method. I think it should log a `WARN` level message